### ISSUE TITLE
Unconnected line

### ISSUE THREAD

**Issue by birgits** [openego/eDisGo#135]
@nesnoj In a few grids there seems to be an unconnected line after the generator import. I stumbled upon it when I tried to find out which feeder all new and reinforced lines are in. When I try the following, e.g. grids 239 and 1729 will return an unconnected line.

```
for line in edisgo.network.results.equipment_changes.index:
    try:
        line.grid.graph.nodes_from_line(line)
    except:
        unconnected_line = line
```

I tried to figure out where in the code this might happen but could not find anything... The unconnected lines are for both grids MV lines and their geom throws an error. Maybe this might be useful information. 
Do you know what might produce this error? Could you have a look at it?

**Comment by birgits** [comment]
Information on data and versions I used:
* edisgo branch: dev (you can just install that branch in order to get the ego_io and ding0 versions I used)
* ego_io: 0.4.1
* ding0: 0.1.6
* dp version = 0.4.2 (I used model_draft because versioned is currently being updated, therefore you have to change the edisgo db config)
* ding0 grid: grid 239 from rli-server -> flexible data -> ding0 -> ding0_data_v0.1.6 (or my Temp directory)
* code I used:
 ```
from edisgo import EDisGo
edisgo = EDisGo(ding0_grid=os.path.join(
        'data', 'ding0_grids__239_with_w_id.pkl'),
                    worst_case_analysis='worst-case-feedin')
edisgo.import_generators(generator_scenario='nep2035')
```
I hope I didn't forget anything. Thank you @nesnoj for your help!

**Comment by nesnoj** [comment]
is this one still pending?

**Comment by birgits** [comment]
Jup...

**Comment by nesnoj** [comment]
I found the problem:

The originally created line for the geno is deleted because another geno is connected to the same line and therefore it is split:

https://github.com/openego/eDisGo/blob/f8c34bc1316e080a585b4e47098e751bf3273b85/edisgo/grid/connect.py#L526-L584
This changes the id but equipment_changes is not updated!

2 issues here:
1. Removed line is not removed from the equipment_changes but is supposed to be.
2. The two new lines are not added to equipment_changes

Fixed in #147, see PR for further details.
